### ISSUE TITLE
Failed to initialize (Newest Space Engineers Update 10/14/2024)

### ISSUE THREAD

**Issue by NingyMD** [viktor-ferenczi/multigrid-projector#88]
My Error Log:

```
2024-10-14 16:48:44.013 - Thread:   1 ->  Info: Multigrid Projector: Loading client plugin
2024-10-14 16:48:44.104 - Thread:   1 ->  [PluginLoader] [Error] Failed to initialize viktor-ferenczi/multigrid-projector|Multigrid Projector because of an error: System.NotSupportedException: Refusing to load the plugin due to potentially incompatible code changes in the game:
Body of patched method MyProjectorBase.GetObjectBuilderCubeBlock has changed: actual 3bff3fea, expected 66247c3b"
Body of patched method MyProjectorBase.InitializeClipboard has changed: actual 2f55ee2e, expected b7140f1d"
Body of patched method MyProjectorBase.RemoveProjection has changed: actual c0739d3a, expected f6fc8084"
Body of patched method MyProjectorBase.Remap has changed: actual 73db0d9e, expected bce65541"
Body of patched method MyProjectorBase.UpdateAfterSimulation has changed: actual df3f0506, expected 47184779"
Body of patched method MyMechanicalConnectionBlockBase.CreateTopPart has changed: actual 439d944c, expected 192978a5"
Body of patched method MyMechanicalConnectionBlockBase.RecreateTop has changed: actual fab7476f, expected 322f8da0, 88379421"
Body of patched method MyMechanicalConnectionBlockBase.CreateTopPartAndAttach has changed: actual 7af10869, expected e1d2892d"
```

![image](https://github.com/user-attachments/assets/66bded1f-918f-48d7-b5ec-baa1a0abc30d)

**Comment by NingyMD** [comment]
There is also a Performance Improvements Error. I will submit the logs for the error in a issue there.

**Comment by viktor-ferenczi** [comment]
Both of them have been fixed and released since.

MGP 0.7.4 has been released, which is compatible with game version 1.205.023
